### ISSUE TITLE
Check dataset for asserting i.i.d

### ISSUE THREAD

**Issue by alessandrodicosola** [alessandrodicosola/NAI#16]

**Comment by alessandrodicosola** [comment]
### Deleted
|Where|File|Reason|
|-----|----|------|
|val|5968.92.11|No similar frames found in neither train nor test set

### Moved
|From|To|File|Reason|
|----|--|----|------|
train|val|3668.92.11|Inserted low-brightness frames in validation set|

### Swap
|From|To|File|File|Reason|
|----|--|----|----|------|
test|train|4860.92.11|6243.92.11|Allow test to have frame stretched vertically

**Comment by alessandrodicosola** [comment]
# **Before**

## Testing
![test_grid_old](https://user-images.githubusercontent.com/52713354/151138180-c65fec1f-ec4d-4e8d-a8ae-0bcdb9fb4b0f.png)

## Validation
![val_grid_old](https://user-images.githubusercontent.com/52713354/151138195-c5541fef-3368-429e-bfc2-d9faa6bd6e78.png)

## Training
![train_grid_old](https://user-images.githubusercontent.com/52713354/151138184-b2a6d6ea-d33e-4fd8-b711-f6420f4999df.png)

**Comment by alessandrodicosola** [comment]
# **After**

## Testing
![test_grid](https://user-images.githubusercontent.com/52713354/151138407-d2c3e143-4ab0-44df-8792-d893d0b61ff6.png)

## Validation
![val_grid](https://user-images.githubusercontent.com/52713354/151138455-e876605b-dd88-4748-94a6-7999292e71ca.png)

## Training
![train_grid](https://user-images.githubusercontent.com/52713354/151138413-c2a92196-5598-4ac8-9498-62745ae69da0.png)

**Comment by alessandrodicosola** [comment]
# **After**

## Testing
![test_grid](https://user-images.githubusercontent.com/52713354/155131110-a5f314c8-7b63-4de8-a5ec-052b6ff6e6b8.png)

## Validation
![val_grid](https://user-images.githubusercontent.com/52713354/155131123-deb89f89-c4dd-4d03-b7ca-da419b616bea.png)

## Training
![train_grid](https://user-images.githubusercontent.com/52713354/155131119-fae9627c-251b-46f8-8dfd-600a8b23509c.png)
